cpg_pipes/pipeline/pipeline.py

```diff
@@ -815,11 +815,19 @@ def __init__(
             input_provider = CsvInputProvider(to_path(csv_path).open())
 
         if input_provider is not None:
+            # Pulls a set of samples to include based on a dict of filters. 
+            filters = get_config()['workflow'].get('sequence_filters')
```

Comment:
Maybe call it `metadata` filters and filter based on anything in the `meta` dictionary? This way it can be generalised.